packages/eslint-plugin/README.md

```diff
@@ -0,0 +1,29 @@
+# About
+`@kaizen/eslint-plugin-kaizen` is a collection of ESLint rules for Kaizen. It aims to solve some of the following problems:
+- Large-scale migrations of Kaizen packages
+- Encourage Kaizen best practices and anticipate common mistakes
+
+## Install
+Add the package to your project's dependencies:
+
+`yarn add @kaizen/eslint-plugin`
+
+Add the plugin to your Eslint configuration: 
+```
+{
+  plugins: ["@kaizen"],
```

Comment:
Just a thought:
In the same vein as the stylelint-plugin, we started with just a plugin but ended up also exposing a recommended config to extend from.
Do you see that happening with this package?

Reply:
Yeah! I think that would be likely. For now we don't really have any rules but I imagine a future we'd want to use this plugin to encourage best Kaizen practices when working with components.